purchase_update_proposal/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+Allow for any user with only read only grants on Purchase Orders to suggest/ask for modification.
+
+Use cases:
+
+* in case of EDI like an Order Response flow (i.e. UBL) which require to deal with update conditions on purchase lines.
```

Comment:
@bealdav I assume that the EDI flow is your use case; can you explain so that I may understand better how this module could be of use there functionally?

Reply:
> I wonder if it's not possible to use purchase_order_revision for this?

I thought about it. It has few code but it duplicate so many data, that doesn't wish my customer. But I implement a custom revision, but just with an incremental integer, you're right (maybe that I should shared). But the 2 concepts are decoupled.

Reply:
The use case is here:
![Selection_158](https://user-images.githubusercontent.com/1853434/105466701-ed3c4f80-5c94-11eb-9585-d6eb92fb3c75.png)
When I insert order response (Réponse fournisseur UBL) I populated with this module the diff between Order and Order response.
Then my customer check: approve (imply update purchase) or refuse the data of the order response with complete traceability on changes with chatter without duplicate purchase.order